dev-helpers/swagger.json

```diff
@@ -0,0 +1,1043 @@
+{
```

Comment:
Change needed: this file should not be present, we don't store definitions in `dev-helpers` as it would get cluttered very quickly 😄